### DISCUSSION TITLE
The birt report will lose data when two tables are displayed side by side in maximo7.6.1.3

### DISCUSSION THREAD

**Discussion by panyueji1213163com** [eclipse-birt/birt#1909] [Q&A]
Hello, everyone, I hope there are professional technical leaders who can help me look at this problem.Thank you very much!The birt report will lose data when two tables are displayed side by side in maximo7.6.1.3；There are only 22 pieces of 30 data in the report preview, and 30 pieces can be displayed normally if the tables are staggered and not juxtaposed. The specific operation can be viewed in the file.
[BIRT并列两栏显示问题.pdf](https://github.com/user-attachments/files/17001216/BIRT.pdf)
[HX_Clearance_small_Tag (1).pdf](https://github.com/user-attachments/files/17001217/HX_Clearance_small_Tag.1.pdf)
[HX_Clearance_small_Tag (2).pdf](https://github.com/user-attachments/files/17001218/HX_Clearance_small_Tag.2.pdf)
[HX_Clearance_small_Tag.pdf](https://github.com/user-attachments/files/17001219/HX_Clearance_small_Tag.pdf)

**Comment by speckyspooky** [comment]
I have created for test reasons a familiar test case. But the main topic is I didn't know which BIRT verion is integrated in maximo7.6.1.3 and your supplier could added some special features which change the behavior of the birt report.

So in best case you can provide a small example report without external sources to reproduce it may be based on the internal BIRT-cars-model would be the best data case.

Back to my test:

I added according to your side 1 Grind with 2 Columns and 1 Row on each cell I added the same table with the same data.
My table has 46 records. I got a similiar result but not the same result on my side. 
The difference is I have then all data without loose of data.
The same problem was that the page break of the internal tables cause a wrong page breake result.
Expected were 4 pages but I got 5.

The problem on my side was the configured page break on the data table (default: 40 rows then break).
I changed the value to a very high value and then all was fine.

Attached you will finde my example. May be you can test it on your side.

[double_table.zip](https://github.com/user-attachments/files/17006090/double_table.zip)


Please be aware to my hint we can assist or check only the original BIRT.
You won't get Maximo-Support here. For that please contact your provider or IBM directly.

**Reply by panyueji1213163com** [reply]
Hello, thank you very much for your reply. The problem is really caused by paging. Some text users demand more, which leads to a higher height of a line. I'll talk to the users.
